pkg/disttask/importinto/subtask_executor.go

```diff
@@ -132,6 +135,25 @@ func postProcess(ctx context.Context, store kv.Storage, taskMeta *TaskMeta, subt
 		return err
 	}
 	return taskManager.WithNewSession(func(se sessionctx.Context) error {
-		return importer.VerifyChecksum(ctx, &taskMeta.Plan, localChecksum.MergedChecksum(), se, logger)
+		err = importer.VerifyChecksum(ctx, &taskMeta.Plan, localChecksum.MergedChecksum(), se, logger)
+		failpoint.Inject("beforeResetTableMode", func() {})
+		failpoint.Inject("errorWhenResetTableMode", func() {
+			failpoint.Return(errors.New("occur an error when reset table mode to normal"))
+		})
+		err2 := ddl.CreateAlterTableModeJob(domain.GetDomain(se).DDLExecutor(), se, model.TableModeNormal, taskMeta.Plan.DBID, taskMeta.Plan.TableInfo.ID)
+		if err2 != nil {
+			callLog.Warn("alter table mode to normal failure", zap.Error(err2))
+		}
+		if err != nil {
+			if err2 == nil {
+				addResetTableModeTask(taskMeta.JobID)
+			}
+			return err
+		}
+		if err2 != nil {
+			return err2
+		}
+		addResetTableModeTask(taskMeta.JobID)
+		return nil
```

Comment:
```suggestion
		if err2 != nil {
			callLog.Warn("alter table mode to normal failure", zap.Error(err2))
		} else {
		    addResetTableModeTask(taskMeta.JobID)
		}
		
		return err
```

A small question, do we need to return error if checksum passed but alter mode failed? Since cleanup will retry the alter table mode.

Reply:
Since if alter mode failed in post process, it will block DML. And cleanup will not alter mode immediately(maybe have fews of import into tasks, it will cleanup one by one). Seems return error is better for user to know why they cannot run DML. It will be confuse that import into task success but they cannot run DML.

Reply:
> Do you mean we can add retry on post process🤔？

Yes, maybe it's not reasonable to fail the whole task only because we failed to alter mode at last? (If it was designed like this, please ignore this 😂)